package.json

```diff
@@ -28,7 +28,7 @@
     "consola": "^2.15.0",
     "defu": "^3.2.2",
     "fs-extra": "^9.0.1",
-    "tailwindcss": "^1.9.6"
+    "tailwindcss": "^2.0.1-compat"
```

Comment:
With the upgrade to nuxt 2.15, will we still need to depend on the compat version of tailwindcss?

Reply:
No, we don't. Please upgrade tailwindcss to 2.0.3

> Once the rest of your tools have added support for PostCSS 8, you can move off of the compatibility build by re-installing Tailwind and its peer-dependencies using the latest tag: